tap_facebook/__init__.py

```diff
@@ -241,6 +241,7 @@ class AdCreative(Stream):
     doc: https://developers.facebook.com/docs/marketing-api/reference/adgroup/adcreatives/
     '''
 
+    @retry_pattern(backoff.expo, (FacebookRequestError, AttributeError), max_tries=5, factor=5)
```

Comment:
This change is for [TDL-13267](https://jira.talendforge.org/browse/TDL-13267): AdCreative requests that are not retrying on `AttributeError`